src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/ApplicationPropertyPage/CompoundTargetFrameworkValueProvider.cs

```diff
@@ -219,6 +231,7 @@ private async Task<bool> IsWindowsPlatformNeededAsync()
             ConfigurationGeneral configuration = await _properties.GetConfigurationGeneralPropertiesAsync();
             _useWPFProperty = (bool?)await configuration.UseWPF.GetValueAsync();
             _useWindowsFormsProperty = (bool?)await configuration.UseWindowsForms.GetValueAsync();
+            _useWinUIProperty = (bool?)await configuration.UseWinUI.GetValueAsync();
```

Comment:
This code awaits three async operations, however if the first returns a non-null value then the second two calls are wasted.

It also looks like these should be local variables rather than fields. I may be missing some usages of the old two on GitHub, but the new one appears to only be used here at least.

Reply:
Addressed on latest commit, thanks!